common/system/tasks/buzzer_heartbeat.cpp

```diff
@@ -1,25 +1,59 @@
-
 #include "buzzer_heartbeat.h"
 
-#include "hardware_manager.h"
-
 static uint32_t last_time = 0;
-static bool isOn;
+// static bool isOn;
 static int buzzerId_ = 0;
+BuzzerState state = BuzzerState::PYRO;
 
 void buzzerHeartbeat_setBuzzer(int buzzerId) { buzzerId_ = buzzerId; }
 
+void pyroSoundOff(bool pyroContCheck, int& pyroCounter);
+
 // Alternates the buzzer between on and off states if a certain period of time
```

Comment:
(nit) Comment is out of date now. Should also probably be moved to the header file